### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by untallucas**
Besides no one will use it in a cellphone, now 20v is responsive, just because we can. 
Test where you want.

### CODE REVIEW

**Comment by loverajoel** [comment]
I'm sure that more than 70% will be mobile

File: src/components/search/Search.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Search extends Component {
             type: 'text',
             ref: 'searchInput',
             className: 'input-search',
-            placeholder: 'Just type a song name and press Enter'
+            placeholder: 'Type a song name and select an option'
```
Comment (Cosmitar) [inline src/components/search/Search.js]: this is for another topic but it's ok :smile:

Comment (untallucas) [inline src/components/search/Search.js]: Yep. Didn't want to open a new branch for this. Changes in app functionality where made, and nobody checked with the copywritting team.
It's an UX matter anyway. If we change the title from "Responsive" to "UX Issues" we are ok. :P

**Comment by Cosmitar** [comment]
On landscape the footer overlaps the search input box.

responsive-bug-01

On landscape the band looks cut at both sides when animates. Also the video title is out of screen.

responsive-bug-02

**Comment by untallucas** [comment]
Landscape? Que no habian cerrado ese antro?

**Comment by Cosmitar** [comment]
hummm, only portrait...

**Comment by untallucas** [comment]
Responsive in portrait devices fixed (in theory). Check it.

**Comment by loverajoel** [comment]
:shipit:

**Pull request merged**